doc/source/contribution/documentation.md

```diff
@@ -49,6 +45,21 @@ Clone Mroonga repository to working directory. Don't forget to do "Git configura
 % git remote add upstream git@github.com:mroonga/mroonga
 ```
 
+### Required softwares
+
+Generating Mroonga documentations needs [Sphinx](https://www.sphinx-doc.org/) and MySQL's source and build directories.
+Instructions for installing Sphinx are provided below.
+Please see {doc}`/install/others` for information on preparing the MySQL source and build directories.
+
+Here are command lines to install Sphinx.
+
+Debian GNU/Linux, Ubuntu:
```

Comment:
If it were just `pip` and `bundle` it would run the same on any OS, so this sentence seems unnecessary.

Reply:
Thanks, you are right. I will fix it now.

Reply:
fix: [f481038](https://github.com/mroonga/mroonga/pull/723/commits/f481038d69213375a0cd2f56a222d1716fb204e3)